cpp/include/logistic_regression.h

```diff
@@ -85,11 +89,12 @@ class LogisticRegression {
     
     for (int i = 0; i < iterations; i++){
       z = x * thetas;
+      z = Sigmoid(z);
```

Comment:
Could you add some comments to explain what z, grad are? I prefer you use a variable named "gradient" in your Fit function so it is easy to understand. Did you finish the matlab code in the class? You could learn about how their algorithms is constructed in the homework.